packages/orchestrator/lib/clients/validate.ts

```diff
@@ -93,7 +93,7 @@ const syncSchema = z.object({
     ...commonSchemaFields,
     payload: syncArgsSchema
 });
-const syncAbortchema = z.object({
+const syncAbortschema = z.object({
```

Comment:
unrelated. typo

Reply:
should it be `syncAbortSchema`? 
```suggestion
const syncAbortSchema = z.object({
```